README.md

```diff
@@ -93,10 +71,44 @@ getPackageName ('package.json')
 
 ### Global Bundle (CDN)
 
-Fluture is hosted in full with all of its dependencies at
-https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.js
+To load Fluture directly into a browser, a code pen, or [Deno][], use one of
+the following downloads from JSDelivr. They are single files that come with all
+of Fluture's dependencies pre-bundled.
+
+- [Fluture Script][]: A JavaScript file that adds `Fluture` to the global
+  scope. Ideal for older browsers and code pens.
+- [Fluture Script Minified][]: The same as above, but minified.
+- [Fluture Module][]: An EcmaScript module with named exports. Ideal for Deno
+  or modern browsers.
+- [Fluture Module Minified][]: The same as above, but minified.
 
-This script will add `Fluture` to the global scope.
+[Fluture Script]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.js
+[Fluture Script Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.min.js
+[Fluture Module]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/module.js
+[Fluture Module Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/module.min.js
```

Comment:
I just realized that jsdelivr strips comments during minification, so the typings reference is lost in the minified version.

Reply:
See https://github.com/jsdelivr/jsdelivr/issues/18224. Depending of what comes out of that, I might need to provide the minified version myself.

Reply:
Not going to resolve this "issue", but just document it, discouraging users from using the minified code with Deno.